src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -210,7 +213,11 @@ export default class ExpressionEnrichmentContainer extends React.Component<
         if (this.isTwoGroupAnalysis && this.props.alteredVsUnalteredMode) {
             columns.push(ExpressionEnrichmentTableColumnType.TENDENCY);
         } else {
-            columns.push(ExpressionEnrichmentTableColumnType.EXPRESSED);
+            if (this.props.isMethylation) {
```

Comment:
again, if we can generalize this rather than have isMethylation, it would be better